.travis.yml

```diff
@@ -1,12 +1,19 @@
 language: java
 
-jdk:
-  - oraclejdk8
-
-addons:
-  apt:
-    packages:
-      - oracle-java8-installer # Updates JDK 8 to the latest available.
+matrix:
+  include:
+# OracleJDK 8
```

Comment:
These comments seem superfluous as it's pretty straightforward based on the name on the next line as to which JDK is being used.

Reply:
Removed.